### DISCUSSION TITLE
Why "prefer-standalone" only on components?

### DISCUSSION THREAD

**Discussion by SamuelT-Beslogic** [angular-eslint/angular-eslint#1667] [General]
So there's a rule `@angular-eslint/prefer-standalone-component` since v16. But why not for Directives, Pipes, etc? Is it still best practice to declare those in modules? Reading https://angular.io/guide/standalone-components I don't see why not.

My current understanding is that I really only need modules if I feel like grouping components/directives/pipes together to avoid having to import multiple in each standalone component (ie: reduced boilerplate). And that otherwise I should go full standalone.

**Comment by JamesHenry** [comment]
It should be the case for everything once this one lands: https://github.com/angular-eslint/angular-eslint/pull/1627#pullrequestreview-1807581472
